### PR TITLE
Turn sourceString() into a paren-less method via global find/replace

### PR DESCRIPTION
**Pull request by geoffjentry**

### CODE REVIEW

File: src/main/scala/cromwell/binding/Task.scala [hunk 1]
```diff
@@ -17,7 +17,7 @@ object Task {
      * For example having a command line of `./script ${File x} ${String x}` is conflicting.
      */
     val commandParameters = ast.findAsts(AstNodeName.CommandParameter)
-    val parameterNames = commandParameters.map { _.getAttribute("name").sourceString() }.toSet
+    val parameterNames = commandParameters.map { _.getAttribute("name").sourceString }.toSet
```
Comment (mcovarr) [inline src/main/scala/cromwell/binding/Task.scala]: parens or lose the dot

File: src/main/scala/cromwell/binding/AstTools.scala [hunk 2]
```diff
@@ -66,7 +66,7 @@ object AstTools {
 
   implicit class EnhancedAstSeq(val astSeq: Seq[Ast]) extends AnyVal {
     def duplicatesByName: Seq[Ast] = {
-      astSeq.groupBy(_.getAttribute("name").sourceString()).collect({case (_ ,v) if v.size > 1 => v.head}).toVector
+      astSeq.groupBy(_.getAttribute("name").sourceString).collect({case (_ ,v) if v.size > 1 => v.head}).toVector
```
Comment (mcovarr) [inline src/main/scala/cromwell/binding/AstTools.scala]: parens and curlies?

File: src/main/scala/cromwell/binding/Workflow.scala [hunk 3]
```diff
@@ -12,7 +12,7 @@ object Workflow {
       Option(call.getAttribute("alias")).getOrElse(call.getAttribute("task"))
     }
 
-    callNames.groupBy {_.sourceString()}.foreach {
+    callNames.groupBy {_.sourceString}.foreach {
```
Comment (mcovarr) [inline src/main/scala/cromwell/binding/Workflow.scala]: dots and curlies don't mix

**Comment by geoffjentry** [comment]
@mcovarr Turnabout is indeed fair play although this was a global find/replace. I'm more than happy to make this a more general refactor PR

**Comment by mcovarr** [comment]
I'd be fine with just cleaning up the area around the changes, the remainder can be tech debt

**Comment by geoffjentry** [comment]
@mcovarr done. although now the breadth of the git diffs are larger so perhaps there are more things :)

File: src/main/scala/cromwell/binding/Workflow.scala [hunk 4]
```diff
@@ -44,8 +44,8 @@ case class Workflow(name: String, declarations: Seq[Declaration], calls: Seq[Cal
    *         inputs that the user needs to provide to this workflow
    */
   def inputs: Seq[WorkflowInput] = {
-    val callInputs = for {call <- calls; input <- call.unsatisfiedInputs} yield input
-    val declarationInputs = for(declaration <- declarations; input <- declaration.asWorkflowInput) yield input
+    val callInputs = for { call <- calls; input <- call.unsatisfiedInputs } yield input
+    val declarationInputs = for { declaration <- declarations; input <- declaration.asWorkflowInput } yield input
```
Comment (mcovarr) [inline src/main/scala/cromwell/binding/Workflow.scala]: Personally I'm not a fan of semicolon-separating in for comprehensions, do the style police have an opinion on this?

Comment (geoffjentry) [inline src/main/scala/cromwell/binding/Workflow.scala]: I don't really like this single line structure either but *shrug*

**Comment by mcovarr** [comment]
:+1: Pretending I didn't see those multiline HOFs

**Comment by mcovarr** [comment]
I was not an officially designated reviewer so I'm not presuming to nominate a second.  :smile:

**Comment by geoffjentry** [comment]
@mcovarr I thought you were captain multi-line HOF? 

re: reviewers I was just going to wait until the end of the sprint in order to make sure it doesn't cause any rebase, even though initially it was such a simple change that it'd seem unlikely. Call yourself a reviewer and then designate one tomorrow or thursday :)

**Comment by geoffjentry** [comment]
FWIW the "why" here, I got tired of IntelliJ incorrectly telling me to call it w/ parens in some spots since it shouldn't be a paren'd method in the first place

**Comment by mcovarr** [comment]
@scottfrazer Have at it

**Pull request merged**